src/_pytest/config/__init__.py

```diff
@@ -256,6 +256,7 @@ def directory_arg(path: str, optname: str) -> str:
 
 builtin_plugins = set(default_plugins)
 builtin_plugins.add("pytester")
+builtin_plugins.add("pytester_assertions")
```

Comment:
```suggestion
builtin_plugins = {*default_plugins, "pytester"}
```

the pytester_assertions module doesn't contain any hook functions and:

> A plugin contains one or multiple hook functions. 

so it shouldn't need to be added to the builtin_plugins

Reply:
That definition is a bit strict I guess, there is nothing actually requiring that a plugin have hooks...

In this case I need it to be a plugin so it gets rewritten.